qutebrowser/browser/browsertab.py

```diff
@@ -196,6 +196,10 @@ def next_result(self, *, result_cb=None):
         """
         raise NotImplementedError
 
+    def searching(self):
+        """Return True if we are currently searching or not."""
+        raise NotImplementedError
+
```

Comment:
Sounds good to me - don't worry about the other one though, I'll add it later when needed.